CHANGELOG.md

```diff
@@ -1,3 +1,436 @@
+## v0.3.0 - 2020-07-29
+
+We're happy to announce the release of Lokomotive v0.3.0 (Coast Starlight).
+
+This release packs new features and bugfixes. Some of the highlights are:
+
+* Kubernetes 1.18.6
+* For Lokomotive clusters running on top of AKS, Kubernetes 1.16.10 is installed.
+* [Component updates](#component-updates)
+
+### Changes in v0.3.0
+
+#### Kubernetes updates
+
+- Update Kubernetes to
+[v1.18.6](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#v1186)
+([#726](https://github.com/kinvolk/lokomotive/pull/726)).
+
+#### Platform updates
+
+##### Packet
+- Update default machine type from `t1.small.x86` to `c3.small.x86`, since
+    `t1.small.x86` are EOL and no longer available in new Packet projects
+    ([#612](https://github.com/kinvolk/lokomotive/pull/612)).
+
+  If you haven't explicitly defined the controller_type and/or
+  worker_pool.node_type configuration options, upgrading to this release will
+  replace your controller and/or worker nodes with c3.small.x86 machines. To
+  avoid this, set these configuration options to the desired values.
+
+##### AKS
+
+- Update Kubernetes version to 1.16.10
+    ([#712](https://github.com/kinvolk/lokomotive/pull/712)).
+
+#### Component updates
+
+- openebs: update to 1.11.0 ([#673](https://github.com/kinvolk/lokomotive/pull/673)).
+
+- calico: update to
+  [v3.15.0](https://docs.projectcalico.org/release-notes/#v3150)
+  ([#652](https://github.com/kinvolk/lokomotive/pull/652)).
+
+#### UX
+
+- prometheus-operator: Organize prometheus related variables under a
+  `prometheus` block in the configuration
+  ([#710](https://github.com/kinvolk/lokomotive/pull/710)).
+
+- Use `prometheus.ingress.host` to expose Prometheus instead of
+    `prometheus_external_url`
+    ([#710](https://github.com/kinvolk/lokomotive/pull/710)).
+
+- contour: Remove `ingress_hosts` from contour configuration
+    ([#635](https://github.com/kinvolk/lokomotive/pull/635)).
+
+#### Features
+
+- Add `enable_toolbox` variable to [rook-ceph
+component](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference/components/rook-ceph)
+([#649](https://github.com/kinvolk/lokomotive/pull/649)). This allows managing
+and configuring Ceph using toolbox pod.
+
+- Add `external_labels` for federated clusters. This helps to identify identify
```

Comment:
i think we should mention that this is a prometheus feature.

Reply:
```suggestion
Add Prometheus feature `external_labels` for federated clusters to Prometheus
    operator component. This helps to identify identify metrics queried from
    different clusters.
```